exercises/nth-prime/nthprime.jl

```diff
@@ -0,0 +1,3 @@
+function main(input)

```

Comment:
```suggestion
function prime(n)
```

See the earlier comments.